pipelines/rhtap-service-push/rhtap-service-push.yaml

```diff
@@ -285,6 +285,26 @@ spec:
           workspace: release-workspace
       runAfter:
         - push-snapshot
+    - name: push-rpm-manifests-to-pyxis
+      taskRef:
+        resolver: "git"
+        params:
+          - name: url
+            value: $(params.taskGitUrl)
+          - name: revision
+            value: $(params.taskGitRevision)
+          - name: pathInRepo
+            value: tasks/push-rpm-manifests-to-pyxis/push-rpm-manifests-to-pyxis.yaml
+      params:
+        - name: pyxisJsonPath
+          value: $(tasks.create-pyxis-image.results.pyxisDataPath)
+        - name: server
+          value: $(tasks.collect-pyxis-params.results.server)
+        - name: pyxisSecret
+          value: $(tasks.collect-pyxis-params.results.secret)
+      workspaces:
+        - name: data
+          workspace: release-workspace
```

Comment:
Shouldn't the new task define the `runAfter` to ensure that it always runs after `create-pyxis-image`?

Reply:
It relies on a `create-pyxis-image` task result so that behavior is inherited for free